### PR TITLE
sync-diff-inspector: support auto-setting snapshot from last syncpoint

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11368
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 1]
```diff
@@ -94,8 +92,16 @@ check-struct-only = false
...
-    # (optional) Uses the snapshot feature. If enabled, historical data is used for comparison
+
+    # (optional) Use TLS to connect TiDB.
+    # security.ca-path = ".../ca.crt"
+    # security.cert-path = ".../cert.crt"
+    # security.key-path = ".../key.crt"
+
+    # (optional) Use the snapshot feature. If enabled, historical data is used for comparison.
     # snapshot = "386902609362944000"
+    # When "snapshot" is set to "auto", the last syncpoint is used for comparison. For details, see <https://github.com/pingcap/tidb-tools/issues/663>.
```
Comment (Leavrth) [inline sync-diff-inspector/sync-diff-inspector-overview.md:103]: the syncpoint is generated by TiCDC.

**Review by Leavrth — APPROVED** [review]

File: sync-diff-inspector/sync-diff-inspector-overview.md (hunk 1 above)
Comment (shichun-0415) [inline sync-diff-inspector/sync-diff-inspector-overview.md:103]:
```suggestion
    # When "snapshot" is set to "auto", the last syncpoint generated by TiCDC is used for comparison. For details, see <https://github.com/pingcap/tidb-tools/issues/663>.
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**